pennylane/templates/subroutines/prepselprep.py

```diff
@@ -221,7 +221,7 @@ def _prepselprep_resources(op_reps, num_control):
         resource_rep(Prod, resources={resource_rep(GlobalPhase): 1, rep: 1}) for rep in op_reps
     )
     return {
```

Comment:
It could be out of the scope of this PR, but should prepselprep and qubitization may support `partial` too?

Reply:
Yes, they should! I thought to add this in #8011 , but it might actually make more sense to add it here!!